sdk/ai/azure-ai-projects/samples/agents/sample_agents_sharepoint.py

```diff
@@ -0,0 +1,78 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_sharepoint.py
+
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the 
+    Sharepoint tool from the Azure Agents service using a synchronous client.
+
+USAGE:
+    python sample_agents_sharepoint.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+
+import os
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import SharepointTool
+
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+
+# Initialize Sharepoint tool with connection id
+sharepoint = SharepointTool(connection_id="sharepoint_connection_name")
+
+# Create agent with Sharepoint tool and process assistant run
+with project_client:
+    agent = project_client.agents.create_agent(
+        model="gpt-4o",
```

Comment:
let's use env variable for model, gpt-4o

Reply:
updated